### PR TITLE
Fixed version of ipaddress dependency

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**

### CODE REVIEW

**Review by etam — DISMISSED** [review]
Please add a comment, why fixed version is required.

**Review by Elfoniok — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -3,7 +3,8 @@ setuptools==34.3.3
 appdirs>=1.4
 click>=4.0
 docker-py==1.7.2
-ipaddress>=1.0.18
+# Fixed version of crossbar dependency.
+ipaddress==1.0.18
```
Comment (chfast) [inline requirements.txt:7]: The `ipaddress` is a part of Python 3.3+, there should not be a reason to install it with pip.

**Review by chfast — CHANGES_REQUESTED** [review]

**Comment by chfast** [comment]
Moreover, you do you need 2 commits to do this single change?

**Comment by mfranciszkiewicz** [comment]
@chfast The point is to maintain change history (i.e. request for a comment). PRs can be squashed.

**Pull request closed without merging**